### PR TITLE
Support \operatorname. Resolve #145

### LINKED ISSUE #145: Support \operatorname
We should support this so you can do, e.g., `\operatorname{span}` (see #144). Its implementation is subtler than it may appear:

http://tex.stackexchange.com/a/84308/9

### PR DESCRIPTION
**Pull request by ronkok**

### CODE REVIEW

**Comment by kevinbarabash** [comment]
operatorname

LaTeX is in green, KaTeX is in red.  Screenshot looks great!

File: src/functions/operators.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+import defineFunction, {ordargument} from "../defineFunction";
+import buildCommon from "../buildCommon";
+import mathMLTree from "../mathMLTree";
+
+import * as html from "../buildHTML";
+import * as mml from "../buildMathML";
+
+// \operatorname
+// amsopn.dtx: \mathop{#1\kern\z@\operator@font#3}\newmcodes@
+defineFunction({
```
Comment (kevinbarabash) [inline src/functions/operators.js]: Thanks for defining these new functions using the new `defineFunction`.  Let me know if you have any ideas of how to make this new structure better.

File: src/functions/operators.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+    mathmlBuilder: (group, options) => {
+        // The steps taken here are similar to the html version.
+        let output = [];
+        if (group.value.value.length > 0) {
+            let temp = mml.buildExpression(group.value.value, options);
+
+            let word = "";
+            for (let i = 0; i < temp.length; i++) {
+                word += temp[i].children[0].text;;
```
Comment (kevinbarabash) [inline src/functions/operators.js]: nit: extra `;`

File: src/functions/operators.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+            }
+            word = word.replace(/\u2212/g, "-");
+            word = word.replace(/\u2217/g, "*");
+            output = [new mathMLTree.TextNode(word)];
+        }
+        const node = new mathMLTree.MathNode("mi", output);
+        node.setAttribute("mathvariant", "normal");
+        return node;
+        // See defineFunction below.
```
Comment (kevinbarabash) [inline src/functions/operators.js]: This comment seems out of place.

File: src/functions/operators.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    },
+});
+
+defineFunction({
+    type: "functionapply",
+
+    // The \operatorname macro expands as: \opNameMain{#1}\functionapply
```
Comment (kevinbarabash) [inline src/functions/operators.js]: Is this is to get around the fact that `mathmlBuilder` only returns a single node?  I wonder if we could have it return a document fragment containing MathML nodes?

Comment (ronkok) [reply src/functions/operators.js]: Correct. I would be happy to get rid of the macro if I knew how to make `mathmlBuilder`  return multiple nodes.

Comment (ronkok) [reply src/functions/operators.js]: Yes, that works out well. I can eliminate the macro entirely.

**Pull request merged**